Utilities/src/main/java/edu/stanford/hivdb/utilities/Cachable.java

```diff
@@ -188,7 +188,7 @@ private boolean loadStaticCache() {
 					InputStream stream = cls
 						.getClassLoader().getResourceAsStream(staticCachePath(f));
 				) {
-					String raw = IOUtils.toString(stream);
```

Comment:
Is deprecated.

Reply:
Use `StandardCharset.UTF8`. However I think I also fixed it in NGS branch (doesn't matter, we can merge them later).

Reply:
Great. Done.